blog/r-tidyfinance-0-1-0/index.qmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "tidyfinance 0.1.0"
+author:
+  - name: Christoph Scheuch
+    url: https://christophscheuch.github.io/
+    affiliations:
+      - name: wikifolio Financial Technologies AG
+  - name: Stefan Voigt
+    url: https://www.voigtstefan.me/
+    affiliations:
+      - name: University of Copenhagen
+      - name: Danish Finance Institute
+  - name: Patrick Weiss
+    url: https://sites.google.com/view/patrick-weiss
+    affiliations:
+      - name: Reykjavik University
+      - name: WU Vienna University of Economics and Business
+date: "2023-03-07"
+description: tidyfinance 0.1.0 is now on CRAN. Discover what this release includes. 
+image: thumbnail.png
+image-alt: An image depicting a busy warehouse scene with a single cardboard box on a conveyor belt, labeled 'tidyfinance' and no other text or stickers. In the background, workers in safety vests are sorting and scanning other packages amidst shelves filled with various boxes and logistics equipment, under a well-lit, organized, and efficient atmosphere. Created with DALL-E 3.
+categories: 
+  - Data
+  - R
+--- 
+
+We are happy to annouce the initial release of the `tidyfinance` R package [on CRAN](https://cran.r-project.org/web/packages/tidyfinance/index.html). The package contains a set of helper functions for empirical research in financial economics, addressing a variety of topics covered in [Tidy Finance with R](../../r/index.qmd) (TFWR). The package is designed to provide shortcuts for issues extensively discussed in the book, facilitating easier application of its concepts. If you want to inspect the details of the package or propose new features, feel free to visit the package repository on [Github](https://github.com/tidy-finance/r-tidyfinance).
```

Comment:
"The package is designed to provide shortcuts for issues extensively discussed in the book, facilitating easier application of its concepts."

Maybe: 
We designed the package to provide easy shortcuts for the applications we discuss in the book.